### ISSUE TITLE
Customizing the template

### ISSUE THREAD

**Issue by psteinroe** [rossmartin/ionic2-alpha-scroll-example#1]
Hi!
Again thanks for this awesome list!
I am trying to add pictures, a header and a description to the items, but it does not work as expected (for example the text always goes under the picture). Is there a way to achieve this like it is possible with the ion-list / ion-item ? 
I tried it inside a div with standard html and using the ion-item / ion - avatar etc..

Would be so great!

Thanks!

**Comment by rossmartin** [comment]
Thanks for reporting I should have time later tonight to look into this.

**Comment by rossmartin** [comment]
I see what's going on here and I know how to make it more flexible.  

If you use this template below it works (don't wrap it in a div that is the problem) - 

``` javascript
alphaScrollItemTemplate: string = `
  <ion-avatar item-left>
    <ion-img src="http://ionicframework.com/dist/preview-app/www/img/avatar-han.png"></ion-img>
  </ion-avatar>
  <h2>Hann</h2>
  <h3>Look, kid...</h3>
  <p>I've got enough on my plate as it is, and I...</p>
`;
```

<img width="287" alt="screen shot 2016-08-24 at 3 34 13 pm" src="https://cloud.githubusercontent.com/assets/2498502/17948950/6b593346-6a10-11e6-91ca-fb4d91617091.png">

What I plan on doing is allowing you to provide `<ion-item>` to the `template` property and this will provide more flexibility (and without duplicating styles).  I should have this updated tonight.

**Comment by rossmartin** [comment]
I fixed this in 1.0.4 by https://github.com/rossmartin/ionic2-alpha-scroll/commit/2d13619472f5d915ad923fd1e4110014bab52447

Now you pass an `<ion-item>` to the `itemTemplate`.
